driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudHelper.java

```diff
@@ -339,8 +335,13 @@ OperationResult executeUpdateOne(final BsonDocument operation) {
                     throw new UnsupportedOperationException("Unsupported argument: " + cur.getKey());
             }
         }
-        return resultOf(() ->
-                toExpected(collection.updateOne(filter, update, options)));
+        BsonDocument filter = arguments.getDocument("filter");
```

Comment:
Nit: In all the other methods the required arguments are pulled out before the options.  Consider whether we should be consistent about that.

Reply:
Done.